build/NuSpecs/AppxManifest.xml

```diff
@@ -1,11 +1,11 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Package xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10" xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10" xmlns:build="http://schemas.microsoft.com/developer/appx/2015/build" IgnorableNamespaces="uap build">
-  <Identity Name="Microsoft.ProjectReunion.0.5" ProcessorArchitecture="x64" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" Version="0.52102.3000.0" />
+  <Identity Name="Microsoft.WindowsAppSDK.0.5" ProcessorArchitecture="x64" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" Version="0.52102.3000.0" />
   <Properties>
     <Framework>true</Framework>
-    <DisplayName>Microsoft.ProjectReunion.0.5</DisplayName>
+    <DisplayName>Microsoft.WindowsAppSDK.0.5</DisplayName>
```

Comment:
You're right, Display Names should be "Windows App SDK..." (I've seen both with and without Framework).

But this is actually bogus. It's not the manifest that ships in anything. It's just metadata in the Transport package as input to the aggregator pipeline. The filename's a misnomer, it should be called TransportMetadata.xml and the schema is a misnomer as it needs to comply with the Transport Package schema not MSIX (the former's been a subset of the latter to date). The only info needed in here is the WinRT inprocessserver extension.

That's also why the `<TargetDeviceFamily...` citing Universal (instead of Desktop) and MaxVersionTested=19H1 haven't been a problem, because they're never used.

There's ES work to refine the Transport package process that'll revise this. And not a moment too soon to prevent such confusion in the future.